giskard/core/suite.py

```diff
@@ -94,9 +92,10 @@ def _repr_html_(self):
         widget = TestSuiteResultWidget(self)
         return widget.render_html()
 
-    def to_mlflow(self, mlflow_client: MlflowClient = None, mlflow_run_id: str = None):
+    def to_mlflow(self, mlflow_client=None, mlflow_run_id: str = None):
```

Comment:
Not sure if it's worth checking the the TYPE_CHECKING constant to keep this anotation: https://docs.python.org/3/library/typing.html#typing.TYPE_CHECKING

Reply:
Did not know about that ! :o will have a look at it